dlt/destinations/impl/redshift/redshift.py

```diff
@@ -76,15 +78,23 @@ def run(self) -> None:
         credentials = ""
         if self._staging_iam_role:
             credentials = f"IAM_ROLE '{self._staging_iam_role}'"
+            if self._region_name:
+                credentials += f" REGION '{self._region_name}'"
```

Comment:
@yannik207 it should be possible to use `self._staging_credentials.region_name`n here and not have to introduce a new setting, no? You should have this already set to the correct region since you need it to upload the files to the staging destination in the first place. So from my point of view we only need to add these two lines but insert from a different place. Let me know if I am missing something here.

Reply:
PS: Adding the lines below also would be good.